apps/transmitmedia.cpp

```diff
@@ -1045,15 +1055,14 @@ class UdpTarget: public Target, public UdpCommon
             cerr << "\nWARN Host for UDP target is not provided. Will send to localhost:" << port << ".\n";
 
         Setup(host, port, attr);
-        if (adapter != "")
+        if (is_multicast && interface_addr.isany() == false)
```

Comment:
I think `== false` makes too much noise.

```suggestion
        if (is_multicast && !interface_addr.isany())
```